.devcontainer/devcontainer.json

```diff
@@ -27,6 +27,7 @@
 	},
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
-		"ms-python.python"
+		"ms-python.python",
+		"ms-vscode-remote.remote-containers"
```

Comment:
I know this is a WIP PR, so feel free to ignore some of my comments in general :).  Just wondering though, is this here intentionally?  It would be nice functionality to have, but we might want to add it as a separate PR along with some updates to the docs on how to use it if so.

Reply:
Yeah, I'll remove this from this PR on the next commit.